frontend/src/constants/datasets/US_mental_health.json

```diff
@@ -5,6 +5,10 @@
     "name": "US Wide Mental Health",
     "hint": "us_wide_mental_health_test",
     "usaWide": true,
+    "demographics": {
```

Comment:
i think demographics is a bit ambiguous. maybe defaultDemographics?

Reply:
startingDemographics?